src/dev/encoder.cpp

```diff
@@ -61,48 +83,43 @@ void Encoder::update() {
 }
 
 int8_t Encoder::readPinValues() {
-    bool aPos = (bool) a.readPin();
-    bool bPos = (bool) b.readPin();
-    int8_t newPos = convertPinValuesToPosition(aPos, bPos);
+    bool aPos = (bool)a.readPin();
+    bool bPos = (bool)b.readPin();
+    int8_t newPos = convertPinValuesToPosition(aPos,bPos);
     return newPos;
 }
 
 int8_t Encoder::convertPinValuesToPosition(bool a, bool b) {
-    if (a == 0 && b == 0) {
-        return 0;
-    } else if (a == 1 && b == 0) {
-        return 1;
-    } else if (a == 1 && b == 1) {
-        return 2;
-    } else if (a == 0 && b == 1) {
-        return 3;
-    }
+    if (a == 0 && b == 0) {return 0;}
```

Comment:
I may be wrong but I think our formatting for most repositories follows along the idea that if statements should be expanded out. If not expanded at least a heavy use of whitespace. But I my be wrong